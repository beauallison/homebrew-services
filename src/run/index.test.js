jest.mock('child_process');

const run = require('./');

describe('run()', () => {
  it('should run a command', async () => {
    const output = await run(['list']);
    expect(output).toMatchSnapshot();
  });
});
