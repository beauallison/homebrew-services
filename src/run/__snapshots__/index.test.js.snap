// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`run() should run a command 1`] = `
"Name          Status  User        Plist
cassandra     stopped
chronograf    stopped
elasticsearch stopped
influxdb      stopped
kafka         stopped
kapacitor     stopped
mongodb       stopped
rabbitmq      stopped
redis         started osx /Users/osx/Library/LaunchAgents/homebrew.mxcl.redis.plist
zookeeper     stopped
"
`;
